src/com/hotmail/kalebmarc/textfighter/main/Weapon.java

```diff
@@ -230,7 +237,25 @@ public void dealDam() {
 	        if (Enemy.get().getHealth() <= Enemy.get().getHealthMax() / 3){
 	            Enemy.get().useFirstAidKit();
 	        }
-        } 
+        }
+        damageDealt = 0;
+    }
+
+    public void criticalHit() {
+
+        if (Random.RInt((int) (100 / this.critChanceMultiplier)) == 1) {

```

Comment:
I would consider pulling this formula out into a separate `private function`, just to make it a little more clear to understand what's going on with this line.

something like this makes it immediately obvious what the code does
```java
if(wasCriticalHit()) {
```